### PR TITLE
Add com.gitlab.JakobDev.jdTextEdit

### PR DESCRIPTION
**Pull request by JakobDev**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

network is required for the Pluginmanager.
cups is required for printing.

### CODE REVIEW

**Comment by JakobDev** [comment]
bot, build com.gitlab.JakobDev.jdTextEdit

File: com.gitlab.JakobDev.jdTextEdit.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --socket=cups
+  - --device=dri
+  - --share=ipc
+
+modules:
+  - pyqt6.yml
+  - qscintilla-pyqt6.yml
```
Comment (hfiguiere) [inline com.gitlab.JakobDev.jdTextEdit.yml:16]: why splitting these? It's harder to maintain. (including enchant)

Comment (JakobDev) [reply com.gitlab.JakobDev.jdTextEdit.yml:16]: I personally find it easier to maintain

Comment (barthalion) [reply com.gitlab.JakobDev.jdTextEdit.yml:16]: Please merge them back into single file. The usual convention is to maintain everything in a single manifest, unless a specific module is very complicated to build or has many dependencies.

**Comment by JakobDev** [comment]
bot, build com.gitlab.JakobDev.jdTextEdit

**Comment by JakobDev** [comment]
I had now put enchant and Qscintilla in the main manifest, but I still think it's better maintainable when it is in different files.

**Comment by nedrichards** [comment]
I don't think enchant is loading dictionaries properly from the error messages I can see and in English the default open buffer should be spelt "Untitled" (this isn't a flathub blocker, just friendly advice to make the app better). Other than that, lgtm

**Comment by JakobDev** [comment]
> I don't think enchant is loading dictionaries properly from the error messages I can see

Could you please explain that? I get no error message and when I enable Spellchecking at Edit>Settings>Spell check everything works as expected.

> in English the default open buffer should be spelt "Untitled" 

Thanks for reporting. Will be fixed in the next release.

**Comment by barthalion** [comment]
Yeah, spellchecking also worked for me.

/merge

**Pull request closed without merging**